src/controllers/scriptController.js

```diff
@@ -56,6 +56,24 @@ const scriptController = {
       });
   },
 
+  deleteScript: (req, res, next) => {
+    const { title } = req.params;
+    // delete the script from the db
+    scriptRepo.deleteScript(title)
+      .then((filename) => {
+        // remove the local copy of the script
+        const path = process.env.UPLOADPATH + filename
```

Comment:
UPLOADPATH doesn't necessarily end with a `/`. In general, it's a good idea to put `/` between path segments so we're always constructing a valid path.